python/ray/dashboard/modules/reporter/gpu_providers.py

```diff
@@ -33,6 +36,7 @@ class ProcessGPUInfo(TypedDict):
 
     pid: int
     gpu_memory_usage: Megabytes
+    gpu_utilization: Optional[Percentage]
```

Comment:
For line 51, we should change `processes_pids: Optional[List[ProcessGPUInfo]]` to `processes_pids: Optional[Map[int, ProcessGPUInfo]]` for laster lookup later.